src/app/compiler.js

```diff
@@ -153,10 +155,11 @@ function Compiler (handleImportCall) {
       compilationFinished({error: 'Compiler not yet loaded.'})
     })
 
-    var newScript = document.createElement('script')
```

Comment:
It seems that jquery removes script tags from the dom after they were executed. So this is the difference between the two methods, but I don't know if that matters. Can you test the code with firefox?
The iframe example cannot be compared to this, I think.